app/experimenter/base/__init__.py

```diff
@@ -18,3 +21,22 @@ def app_version():
             app_version = ""
 
     return app_version
+
+
+@cache
+def get_uploads_storage():
+    uploads_storage_cls = get_storage_class(settings.UPLOADS_FILE_STORAGE)
+    if settings.UPLOADS_FILE_STORAGE == "django.core.files.storage.FileSystemStorage":
+        return uploads_storage_cls(
+            location=settings.UPLOADS_FILE_ROOT, base_url=settings.UPLOADS_FILE_BASE_URL
+        )
+    elif settings.UPLOADS_FILE_STORAGE == "storages.backends.gcloud.GoogleCloudStorage":
+        return uploads_storage_cls(
+            project_id=settings.UPLOADS_GS_PROJECT_ID,
+            bucket_name=settings.UPLOADS_GS_BUCKET_NAME,
+            credentials=service_account.Credentials.from_service_account_info(
+                json.loads(settings.UPLOADS_GS_CREDENTIALS)
+            ),
+        )
```

Comment:
Could you do this by just checking if those settings are set and initializing the correct one, ie check if the gcp ones are set and initialize gcp otherwise fall back to localfile, rather than doing a string comparison?  I know those aren't likely to change I just don't like hardcoding those long strings in multiple places.